conf/modules.config

```diff
@@ -218,7 +218,7 @@ if (params.aligner == 'kallisto') {
 if (params.aligner == 'cellrangermulti') {
     process {
         withName: FASTQC { ext.prefix = { "${meta.id}_${meta.feature_type}" } } // allow distinguishment of data types after renaming
-        withName: 'NFCORE_SCRNASEQ:SCRNASEQ:CELLRANGER_MULTI_ALIGN:CELLRANGER_MULTI' {
+        withName: 'NFCORE_SCRNASEQ:SCRNASEQ:CELLRANGER_MULTI_ALIGN:CELLRANGER_MULTI|NFCORE_SCRNASEQ:SCRNASEQ:CELLRANGER_MULTI_ALIGN_VDJ:CELLRANGER_MULTI_DEMUX|NFCORE_SCRNASEQ:SCRNASEQ:CELLRANGER_MULTI_ALIGN_VDJ:CELLRANGER_MULTI_IMMUNE' {
```

Comment:
I tried:

`withName: 'NFCORE_SCRNASEQ:SCRNASEQ:CELLRANGER_MULTI_ALIGN*' {`

But it didn't work for me?

Reply:
I think it's a regex, not a glob, so `.*` should work.